.gitignore

```diff
@@ -351,4 +351,4 @@ config/rbac/role.yaml
 bundle/
 bundle.Dockerfile
 
-/bin
\ No newline at end of file
+/bindatabase
```

Comment:
the folder I see locally it's called just `database`:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	database/ 
```

Reply:
ah right! thanks for spotting that mistake 🤦‍♂️ 
I ran `echo database >> .gitignore` but I did not realize that the previous (last) line did not have a trailing `\n`!